### PR TITLE
Image upload be fe

### PR DESCRIPTION
**Pull request by mariothomsen**
FE:
- Added tags
- Created fetch to send images to be and fetch tags as response

BE:
- Resetup complete backend
- Save images in db
- Response tags from be to fe

Stories:
https://github.com/mariothomsen/keepit/issues/6
https://github.com/mariothomsen/keepit/issues/4
https://github.com/mariothomsen/keepit/issues/6

### CODE REVIEW

File: backend/src/Repository/ImageRepository.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+class ImageRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, Image::class);
+    }
+
+    public function save(Image $image): Image  {
+        // $this->_em == getEntityManager()
+        $this->_em->persist($image);
```
Comment (tine-krueger) [inline backend/src/Repository/ImageRepository.php:24]: Den Kommentar kannst Du löschen

File: backend/src/Utils/LocalFiles.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            }
+        } else {
+            throw new \Exception('did not match data URI with image data');
+        }
+        $file = UPLOAD_DIR . uniqid() . '.' . $type;
+        file_put_contents($file , $data);
+        return $file;
+    }
+}
+
```
Comment (tine-krueger) [inline backend/src/Utils/LocalFiles.php:29]: nice one, would like to talk to you, as I may need Image-Uploads as well.

File: frontend/src/Pages/NewKeepitPage.js [hunk 3]
```diff
@@ -1,33 +1,54 @@
...
-      <h1>Page: New Image</h1>
-      {images.map((image, index) => (
-        <div key={index}>
-          <img src={image['data_url']} alt="" width="100" />
-          <button onClick={() => remove(index)}>Remove</button>
-        </div>
+      <h1>Page: New</h1>
+      {images &&
+        images.map((image, index) => (
+          <div key={index}>
```
Comment (tine-krueger) [inline frontend/src/Pages/NewKeepitPage.js:45]: Do really want to use index here?

File: backend/src/Utils/LocalFiles.php (hunk 2 above)
Comment (mariothomsen) [reply backend/src/Utils/LocalFiles.php:29]: Sure, just write me on slack.

File: frontend/src/Pages/NewKeepitPage.js (hunk 3 above)
Comment (mariothomsen) [reply frontend/src/Pages/NewKeepitPage.js:45]: Good point, I will try it with the base64 image-sources. Should be unique. The key for tags below is missing as well. Thx!

Comment (mariothomsen) [reply frontend/src/Pages/NewKeepitPage.js:42]: Update: I need the index for handling the delete button either way, but keep this in mind.

**Pull request merged**